lib/plugin.dart

```diff
@@ -299,6 +302,15 @@ class AtomDartPackage extends AtomPackage {
         'order': 1
       },
 
+      // debugger
+      'debugBrowserLocation': {
+        'title': 'Debug Browser',
+        'description': 'The location of a Browser with Chrome DevTools Protocol.',
```

Comment:
The debug protocol seems to be used by more than one browser, according to:
https://github.com/cyrus-and/chrome-remote-interface#implementations
I'm happy either way.